packages/api/es/collections.js

```diff
@@ -27,19 +28,35 @@ class Collection extends BaseSearch {
           }
         },
         aggs: {
-          hashes: {
-            terms: {
-              field: '_uid'
+          collections: {
+            filter: {
+              term: {
+                _type: 'collection'
+              },
             },
             aggs: {
-              stats: {
-                children: {
-                  type: 'granule'
+              name: {
+                terms: {
+                  field: 'name'
                 },
                 aggs: {
-                  count: {
+                  version: {
                     terms: {
-                      field: 'status'
+                      field: 'version'
+                    },
+                    aggs: {
+                      stats: {
+                        children: {
+                          type: 'granule'
```

Comment:
Reviewers - note I didn't include the size de-limiter from #1883.    That's up for debate, although given this is fixing a prod issue safer is *probably* better.     That's up for debate in this case.

Reply:
I agree with this decision

Reply:
So I would say that is good for 2.0.6. I'm guessing it's passing the tests then that I put in for >10 granules on master?? (see https://github.com/nasa/cumulus/pull/1870 and you can probably just close that one or I can once you confirm)

Reply:
@laurenfrederick  if you mean #1883 and the units introduced in #1821 then yes, the master PR is passing all tests.